extensions/aws/controllerservices/AWSCredentialsService.h

```diff
@@ -53,6 +48,25 @@ class AWSCredentialsService : public core::controller::ControllerService {
       : ControllerService(name) {
   }
 
+  EXTENSIONAPI static constexpr const char* Description = "AWS Credentials Management Service";
+
+  EXTENSIONAPI static const core::Property UseDefaultCredentials;
+  EXTENSIONAPI static const core::Property AccessKey;
+  EXTENSIONAPI static const core::Property SecretKey;
+  EXTENSIONAPI static const core::Property CredentialsFile;
+  static auto properties() {
+    return std::array{
+      UseDefaultCredentials,
+      AccessKey,
+      SecretKey,
+      CredentialsFile
+    };
+  }
+
+  EXTENSIONAPI static constexpr bool SupportsDynamicProperties = false;
+  EXTENSIONAPI static constexpr bool SupportsDynamicRelationships = false;
```

Comment:
There should be no need for `SupportsDynamicRelationships` for anything other than Processors. Services can't have relationships.

Reply:
Yeah, it didn't make much sense, but I didn't want to change the existing logic.

Currently, every controller service in the manifest has a `"supportsDynamicRelationships": false` line; should I remove that, or leave it hard-coded to false?

Reply:
Done in 3c2574ae73f8020181503d669161607601f861cf

The manifest will still have a `"supportsDynamicRelationships": false` line in each controller service block, as it did before this PR, I hope that's OK.  We can remove it later, after checking with C2 developers.